admin_manual/configuration_database/linux_database_configuration.rst

```diff
@@ -98,26 +98,24 @@ Please refer to the `page in the MySQL manual <https://mariadb.com/kb/en/library
 
 .. note:: MariaDB is backwards compatible with MySQL.  All instructions work for both. You will not need to replace mysql with anything.
 
-The PHP configuration in :file:`/etc/php7/conf.d/mysql.ini` could look like this:
+Enabling the pdo_mysql extension could be done in various ``php.ini`` files depending on your installation of PHP.
+This could be done within :file:`/etc/php/php7.2/fpm/php.ini` file if you are using php-fpm.
```

Comment:
it's not so much if it is ubuntu, rather than php-fpm or apache module.

Reply:
Yes the main difference would be between php-fpm and the apache module. Since most install with a working config it seems confusing to make suggestions to modify the configs. What would you suggest doing?

Reply:
If we can use default configs in general, we should just focus on things that must or should be changed. Either they must be changed, or they allow a better performance.